addon/factory-guy.js

```diff
@@ -379,7 +379,7 @@ class FactoryGuy {
         fixture     = this.buildRaw(assign(args, {buildType: 'make'}));
 
     if (this.isModelAFragment(modelName)) {
-      return fixture;
+      return this.store.createFragment(modelName, fixture);
```

Comment:
`Store#createFragment` is the API from [ember-data-model-fragments](https://github.com/lytics/ember-data-model-fragments)